sprites/Makefile

```diff
@@ -12,6 +16,9 @@ Pill.json: Pill.raw.json
 Pill.xpm: Pill.png
 	convert Pill.png Pill.xpm
 
+Pill.txt: Pill.png
+	convert Pill.png Pill.txt
```

Comment:
I am not expert on makefiles at all, but won't this rule create a collision with `%.txt`?